default/stringtables/en.txt

```diff
@@ -5429,7 +5429,10 @@ METER_POPULATION_VALUE_LABEL
 Population
 
 METER_POPULATION_VALUE_DESC
-An abstract value for the size of a colony, relative to differing species.
+'''An abstract value for the size of a colony on a planet, relative to differing species. The Population on [[encyclopedia PC_COLONY]] ships is referred to as Colonists. When colonizing a planet, the number of Colonists aboard the ship will be the starting Population on the newly settled planet.
```

Comment:
meaning of abstract and relative are unclear

Reply:
I'd probably put some information about what population does in the first paragraph, and move the info about colony ships later.  eg. many bonuses to resource output depend on population, and

Reply:
Tell me if it's better:

```
'''The Population represents the size of a colony on a planet. Its base value depends of the [[encyclopedia ENC_SPECIES]] inherent characteristics and the planet's [[encyclopedia ENVIRONMENT_TITLE]] suitability for a given species. Many [[metertype METER_INDUSTRY]] and [[metertype METER_RESEARCH]] output bonuses are linked to the Population size: bigger is the number, greater are the bonuses.

A Population of at least three and a minimum [[metertype METER_HAPPINESS]] of 5 is necessary for a colony to act as a source of settlers for colonizing other planets.  When producing a [[encyclopedia PC_COLONY]] ship, it has its own Population referred to as Colonists, and its value fixed according the Colonization capacity of the ship design.  When colonizing a planet, the number of Colonists aboard the ship will be the starting Population on the newly settled planet.'''
```

Reply:
What is the "base value"...? The initial population after colonization, or the target value?

...`depends on the species and how suitable the planet's environment is for that species.`

`: larger population gives more bonuses.`

'A colony ship has a population of colonists, which is determined by the colonzation capacity of the ship design.'